openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/certificate_estimation/HVAC2_ESC_calculation.py

```diff
@@ -93,7 +93,27 @@ def formula(buildings, period, parameters):
                         (heating_capacity * equivalent_heating_hours) / baseline_ACOP
                     ])
 
-  
+class HVAC2_THEC(Variable):
```

Comment:
can we change the names of these variables to something more informative
like HVAC2_thec_or_equation

Reply:
Okay will do - hold the line